.gitignore

```diff
@@ -0,0 +1 @@
+/nbproject/private/
```

Comment:
.. Sometimes i hate this IDE... i dont know what shortcut i pushed that it created this project folder... normaly this shouldnt be there.